packages/google_maps_flutter/google_maps_flutter_web/lib/src/marker_clustering.dart

```diff
@@ -109,7 +114,7 @@ class ClusterManagersController extends GeometryController {
   void _clusterClicked(
       ClusterManagerId clusterManagerId,
       gmaps.MapMouseEvent event,
-      MarkerClustererCluster markerClustererCluster,
+      MarkerClustererCluster<T> markerClustererCluster,
```

Comment:
I'm fine with the clusterers being generics, because at the end of the day they're "a `List<T>` with a bunch of other stuff".

As long as we don't need too many calls to `doOnType` stuff...